cg/constants/gene_panel.py

```diff
@@ -37,6 +37,7 @@ class GenePanelMasterList(StrEnum):
     PANELAPP_GREEN: str = "PANELAPP-GREEN"
```

Comment:
Just a minor thing, there are a few panel names out of alphabetical order starting with PEDHEP, would be nice to fix that. But not strictly necessary